src/main/kotlin/no/nav/familie/ef/sak/api/dto/DokumentinfoDto.kt

```diff
@@ -0,0 +1,11 @@
+package no.nav.familie.ef.sak.api.dto
+
+import no.nav.familie.kontrakter.felles.journalpost.Journalposttype
+import java.time.LocalDate
+
+data class DokumentinfoDto(val dokumentinfoId: String,
+                           val filnavn: String?,
+                           val tittel: String,
+                           val journalpostId: String,
+                           val dato: LocalDate?,
+                           val journalposttype: Journalposttype)
```

Comment:
Nå når jeg sjekket frontend så såg jag verdiene på denne.. Tenker den fint kan returnere bedre verdier :)

Reply:
Men hvor bør grensen gå for dette? Vi har utstrakt bruk av denne verdien allerede. Bør vi lage oss en mapper til vår "egen" journalposttype alle disse stedene? 🤔 Det blir jo litt merkelig å bruke journalposttype "I, U, N" noen steder og "Inngående, Utgående, Notat" andre steder?

Reply:
Jeg tenker att ingen skjønner hva I, U, N, BEH_SAK, JFR, etc betyder, og att vi godt burde bruke verdier som gir noe verdi. Er mye enklere å lese og skjønne koden hvis vi mapper om disse verdienn

Grensen går mellom andre systemer og våre applikasjoner, klienten henter data fra andra systemet, mapper om slik att vi får verdier vi enkelt kan forholde oss til. 
Ulempen med det er ju att det kan være vanskelig å finne ut hvor blir I mappet til Inngående, men tenker fordelene veier opp godt. 

Men er kanskje noe vi burde ta med hele teamet